src/Models/Product.php

```diff
@@ -198,7 +198,7 @@ public function getUrlAttribute(): string
     {
         $configModel = config('rapidez.models.config');
 
-        return '/' . ($this->url_key ? $this->url_key . $configModel::getCachedByPath('catalog/seo/product_url_suffix', '.html') : 'catalog/product/view/id/' . $this->entity_id);
+        return '/' . ($this->url_key ? $this->url_key . $configModel::getValue('catalog/seo/product_url_suffix') : 'catalog/product/view/id/' . $this->entity_id);
```

Comment:
`.html` is the default within Magento: https://github.com/magento/magento2/blob/6340fa15d608c91ee85b75e4f99cf239a0c8e857/app/code/Magento/Catalog/etc/config.xml#L44, our fallback should be inline with that.